components/Common/Breadcrumbs/BreadcrumbRoot/index.tsx

```diff
@@ -0,0 +1,56 @@
+import HomeIcon from '@heroicons/react/24/outline/HomeIcon';
+import type { FC, PropsWithChildren, ComponentProps } from 'react';
+import { useIntl } from 'react-intl';
+
+import BreadcrumbItem from '@/components/Common/Breadcrumbs/BreadcrumbItem';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type BreadcrumbRootProps = {
+  hideHome?: boolean;
+};
+
+const BreadcrumbRoot: FC<PropsWithChildren<BreadcrumbRootProps>> = ({
+  hideHome = false,
+  children,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const navigateToHome = formatMessage({
+    id: 'components.common.breadcrumbs.navigateToHome',
+  });
+
+  return (
+    <nav aria-label="breadcrumb">
+      <ol
+        itemScope
+        itemType="https://schema.org/BreadcrumbList"
+        className={styles.list}
+      >
+        {!hideHome && (
+          <BreadcrumbItem>
+            <LocalizedLink
+              itemScope
+              itemType="http://schema.org/Thing"
+              itemProp="item"
+              itemID="/"
+              href="/"
+            >
+              <HomeIcon
+                {...{ itemProp: 'name' as ComponentProps<'svg'> }}
```

Comment:
I think what you can do is to do something like:

```
const HeroIconAsSVG = HeroIcon as SVGSVGElement;
```

On the beginning of the file. That should do it. Is dirty but less than this, and then you can add an inline comment saying why this was done like

```
// Apparently the type of HeroIcon doesn't support `itemProp` on their type
// @see https://github.com/tailwindlabs/heroicons/issues/XXXX

and you can open an issue on https://github.com/tailwindlabs/heroicons/issues and link it there

Reply:
Let's then revert this and go back to span. Quite interesting. I wonder... why 🤔